joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 100  
vel_controller0:
  type: effort_controllers/JointVelocityController
  joint: shoulder_pan_joint
  pid: {p: 10000.0, i: 0.01, d: 10.0}
vel_controller1:
  type: effort_controllers/JointVelocityController
  joint: shoulder_lift_joint
  pid: {p: 10000.0, i: 0.01, d: 10.0}
vel_controller2:
  type: effort_controllers/JointVelocityController
  joint: elbow_joint
  pid: {p: 10000.0, i: 0.01, d: 10.0}
vel_controller3:
  type: effort_controllers/JointVelocityController
  joint: wrist_1_joint
  pid: {p: 10000.0, i: 0.01, d: 10.0}
vel_controller4:
  type: effort_controllers/JointVelocityController
  joint: wrist_2_joint
  pid: {p: 10000.0, i: 0.01, d: 10.0}
vel_controller5:
  type: effort_controllers/JointVelocityController
  joint: wrist_3_joint
  pid: {p: 10000.0, i: 0.01, d: 10.0}
vel_controller:
  type: effort_controllers/JointTrajectoryController
  joints:
     - shoulder_pan_joint
     - shoulder_lift_joint
     - elbow_joint
     - wrist_1_joint
     - wrist_2_joint
     - wrist_3_joint
  constraints:
      goal_time: 0.6
      stopped_velocity_tolerance: 0.05
      shoulder_pan_joint: {trajectory: 0.1, goal: 0.1}
      shoulder_lift_joint: {trajectory: 0.1, goal: 0.1}
      elbow_joint: {trajectory: 0.1, goal: 0.1}
      wrist_1_joint: {trajectory: 0.1, goal: 0.1}
      wrist_2_joint: {trajectory: 0.1, goal: 0.1}
      wrist_3_joint: {trajectory: 0.1, goal: 0.1}
  stop_trajectory_duration: 0.5
  state_publish_rate:  25
  action_monitor_rate: 10
  pid: {p: 1000.0, i: 0.01, d: 10.0}